tools/goctl/model/sql/parser/parser.go

```diff
@@ -360,6 +360,7 @@ func getTableFields(table *model.Table) (map[string]*Field, error) {
 		}
 
 		field := &Field{
+			NameOriginal:    each.Name,
```

Comment:
Why do you need to add this field and what is the status quo? What are the expectations? Did not understand this pr

Reply:
Because when generating the struct model, judging whether to add a filed tag is judged by the field's NameOriginal variable.
The specific code is in line 27 of **tools/goctl/model/sql/gen/field.go**.
![image](https://user-images.githubusercontent.com/13959374/147954715-14dc419d-afd3-47c9-8b38-a044441044ba.png)

In the current version, it is fine to assign a value to the NameOriginal variable when querying the table structure. 
The db tag is ignored when generating, which causes the column not found error when querying the generated code.
You can use the sql in command_pg_test to create a table and try it out. 
```
CREATE TABLE "public"."users" (
  "id" serial NOT NULL,
  "account" varchar(20) COLLATE "pg_catalog"."default" NOT NULL,
  "avatar" text COLLATE "pg_catalog"."default",
  "nick_name" varchar(60) COLLATE "pg_catalog"."default",
  "register_time" timestamp(6) NOT NULL,
  "update_time" timestamp(6),
  "password" varchar(255) COLLATE "pg_catalog"."default",
  "email" varchar(100) COLLATE "pg_catalog"."default",
  "reset_key" varchar(10) COLLATE "pg_catalog"."default",
  "active" bool NOT NULL DEFAULT true,
  CONSTRAINT "users_pk" PRIMARY KEY ("id")
)
;

ALTER TABLE "public"."users"
  OWNER TO "postgres";
```
The currently generated model and sqlx cannot use the crud method correctly.

Reply:
Get